peekingduck/pipeline/nodes/model/posenet.py

```diff
@@ -22,12 +22,67 @@
 
 
 class Node(AbstractNode):
-    """PoseNet node class that initalises a PoseNet model to detect poses from
-    an image
+    # TODO: Add in glossary for keypoint indices
+    """PoseNet node that initalises a PoseNet model to detect human poses from
+    an image.
+
+    The PoseNet node is capable of detecting multiple human figures
+    simultaneously per inference and for each detected human figure, 17 keypoints
+    are estimated. The keypoint indices table can be found :term:`here <keypoint indices>`.
+
+    Inputs:
+        |img|
+
+    Outputs:
+        |bboxes|
+
+
+        |keypoints|
+
+
+        |keypoint_scores|
+
+
+        |keypoints_conns|
+
+
+
+    Configs:
+        model_type (:obj:`str`): **{"resnet", "50", "75", "101"}, default="resnet"**
+
+            Defines the backbone model for PoseNet.
+
+        weights_dir (:obj:`List`):
+            list of directories pointing to model weights
+
+        blob_file (:obj:`str`):
+            name of file to be downloaded, if weights are not found in `weights_dir`
+
+        model_files (:obj:`Dict`):
+            dictionary pointing to path of model weights file
+
+        resolution (:obj:`Dict`): **default = { height: 225, width: 225 }**
+
+            resolution of input array to PoseNet model
+
+        max_pose_detection (:obj:`int`): **default = 10**
+
+            maximum number of poses to be detected
+
+        score_threshold (:obj:`float`): **[0,1], default = 0.4**
+
+            threshold to determine if detection should be returned
+
```

Comment:
just a thought - for YOLO there's a whole bunch of params related to MODEL_NODE - we can exclude them?

Reply:
yeah i think it will be okay to exclude them. would there be a scenario in which people will change them?

Reply:
Unlikely, let's go with this for now.